lib/adapter.js

```diff
@@ -117,25 +116,25 @@ class KnexAdapter {
     await this.dropTable();
     await this.createTable();
 
-    await this.policies.insert([
+    await this.knex(this.tableName).insert([
```

Comment:
Should make a private method `this.getKnex()` which wraps this logic. Though i'm not going to hold up this PR over that.

Reply:
Ok that was bugging me too but I was feeling lazy about figuring out what to name it. `this._getKnex()` sounds good